nautobot_ssot/integrations/dna_center/diffsync/models/base.py

```diff
@@ -0,0 +1,163 @@
+"""DiffSyncModel subclasses for Nautobot-to-DNA Center data sync."""
+
+from typing import Optional, List
+from uuid import UUID
+from diffsync import DiffSyncModel
+
+
+class Area(DiffSyncModel):
+    """DiffSync model for DNA Center areas."""
+
+    _modelname = "area"
+    _identifiers = ("name", "parent")
+    _attributes = ()
+    _children = {}
+
+    name: str
+    parent: Optional[str] = None
+    buildings: Optional[List["Building"]] = []
```

Comment:
Is it on purpose that hit is neither identifier, nor attribute or child?

Reply:
I think I had at one point had Building as a child but removed that relationship. I should probably just remove the buildings attribute.